src/components/fields/Date/index.ts

```diff
@@ -104,6 +104,7 @@ class DateComponent extends FormComponentBase {
           id=${this.theComponentId}
           name="DateOnly"
           dataTestId=${this.testId}
+          .value=${dateValue}
```

Comment:
Is this needed? Since the `value` property will be overwritten in the `LionInputDatetime ` class anyways?